### PR TITLE
Fix typos in comments

### PR DESCRIPTION
**Pull request by shuoer86**

### CODE REVIEW

File: evm/src/cpu/stack.rs [hunk 1]
```diff
@@ -75,7 +75,7 @@ pub(crate) const JUMPDEST_OP: StackBehavior = StackBehavior {
 
 // AUDITORS: If the value below is `None`, then the operation must be manually checked to ensure
 // that every general-purpose memory channel is either disabled or has its read flag and address
-// propertly constrained. The same applies  when `disable_other_channels` is set to `false`,
+// property constrained. The same applies  when `disable_other_channels` is set to `false`,
```
Comment (Nashtare) [inline evm/src/cpu/stack.rs:78]:
```suggestion
// properly constrained. The same applies  when `disable_other_channels` is set to `false`,
```

**Review by Nashtare — COMMENTED** [review]
Thanks! There's one change that needs a fix

File: evm/src/cpu/stack.rs (hunk 1 above)
Comment (shuoer86) [reply evm/src/cpu/stack.rs]: changed. thank you for your review.

**Comment by shuoer86** [comment]
@Nashtare changed. Thank you for your review.

**Review by Nashtare — APPROVED** [review]

**Pull request merged**